plugins/org.yakindu.base.types.edit/src/org/yakindu/base/types/provider/PropertyItemProvider.java

```diff
@@ -144,10 +144,13 @@ public Object getImage(Object object) {
 	 */
 	@Override
 	public String getText(Object object) {
-		Property variable = (Property)object;
-		StringBuilder builder = new StringBuilder(variable.getName());
-		builder.append(" : ");
-		if(variable.getType() != null)
+		Property variable = (Property) object;
+		StringBuilder builder = new StringBuilder();
+		if (variable.getName() != null) {
```

Comment:
In our case (SCTUnit) we have properties without names and only want to see the types in the Outline View.